pathd/path_pcep_pcc.c

```diff
@@ -766,16 +773,17 @@ void specialize_output_path(struct pcc_state *pcc_state, struct path *path)
 	path->sender = pcc_state->pcc_addr_tr;
 
 	if ((path->originator == NULL)
-	    || (strcmp(path->originator, pcc_state->originator) == 0)) {
+	    || (strcmp(path->originator, pcc_state->originator) == 0)
+	    || pcc_state->pce_opts->is_best) {
 		is_delegated = path->type == SRTE_CANDIDATE_TYPE_DYNAMIC;
 		/* it seems the PCE consider updating an LSP a creation ?!?
 		at least Cisco does... */
 		was_created = path->update_origin == SRTE_ORIGIN_PCEP;
 	}
 
 	path->pcc_id = pcc_state->id;
-	path->go_active = is_delegated;
-	path->is_delegated = is_delegated;
+	path->go_active = (is_delegated & pcc_state->pce_opts->is_best);
```

Comment:
This shouldn't com form there, this should be a field of pcc_state structure.

Reply:
ok, is_best and previous_best moved to pcc_state , not in pce_opts anymore